test/nyc-integration-old.js

```diff
@@ -329,7 +329,53 @@ describe('the nyc cli', function () {
 
         const proc = spawn(process.execPath, args, {
           cwd: fixturesCLI,
-          env: env
+          env
+        })
+
+        let stderr = ''
+        proc.stderr.on('data', function (chunk) {
```

Comment:
Can we drop the `--delete` option from this test so we test just `nyc instrument ./ ./`?  You have a test for blocking in-place delete below.